### ISSUE TITLE
about ppltasks.cpp

### ISSUE THREAD

**Issue by nil-ref** [ntoskrnl7/crtsys#23]
```
C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\winrt\wrl\wrappers\corewrappers.h(98,15): error C2039: "ReleaseMutex": is not a member of '`global namespace`' (compiling source file C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.34.31721\crt\src\stl\ppltasks.cpp) [F:\git\crtsys\build_2022_x64_3\crtsys.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\winrt\wrl\wrappers\corewrappers.h(98,1): error C3861: "ReleaseMutex": identifier not found (compiling source file C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.34.31721\crt\src\stl\ppltasks.cpp) [F:\git\crtsys\build_2022_x64\crtsys.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\winrt\wrl\wrappers\corewrappers.h(111,15): error C2039: "ReleaseSemaphore": is not a member of '`global namespace`' (compiling source file C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.34.31721\crt\src\stl\ppltasks.cpp) [F:\git\crtsys\build_2022_x64\crtsys.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\winrt\wrl\wrappers\corewrappers.h(111,1): error C3861: "ReleaseSemaphore": identifier not found (compiling source file C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.34.31721\crt\src\stl\ppltasks.cpp) [F:\git\crtsys\build_2022_x64\crtsys.vcxproj]
```

When I add ppltasks.cpp to the exclusions, it compiles fine.

**Comment by ntoskrnl7** [comment]
0.1.10
fix: SDK/WDK 10.0.22621.0 and MSVC 14.33.31629 are supported.


- VC Tools 14.34.31721 and Windows SDK 10.0.22621.0 have not been tested yet.

- The ppltasks.cpp file must be compiled as it is used by the Concurrency Support Library (https://en.cppreference.com/w/cpp/thread).

- Since there is no part that uses CreateMutexA/W and CreateSemaphoreA/W/ExA/ExW functions in MS CRT, the method of creating dummy functions for ReleaseMutex and ReleaseSemaphore functions will be applied so that ppltasks.cpp can be built without any problems.

**Comment by ntoskrnl7** [comment]
@nil-ref 

0.1.10
fix: SDK/WDK 10.0.22621.0 and MSVC 14.33.31629 are supported.

VC Tools 14.34.31721 and Windows SDK 10.0.22621.0 have not been tested yet.

The ppltasks.cpp file must be compiled as it is used by the Concurrency Support Library (https://en.cppreference.com/w/cpp/thread).

Since there is no part that uses CreateMutexA/W and CreateSemaphoreA/W/ExA/ExW functions in MS CRT, the method of creating dummy functions for ReleaseMutex and ReleaseSemaphore functions will be applied so that ppltasks.cpp can be built without any problems.
